weblate/static/loader-bootstrap.js

```diff
@@ -1525,8 +1525,11 @@ $(function () {
     $('.search-group li a').click(function () {
         var $this = $(this);
         var $button = $this.closest('.input-group-btn').find('button');
-        $button.data('field', $this.data('field'));
+        $button.attr('data-field', $this.data('field'));
```

Comment:
Why this change? I'm not saying it is bad (I have no clue), but I'm curious for reasons.

Reply:
So, it doesn't affect a lot to be fair. But we have `data-field` mentioned in the DOM of the button. And `.data()` updates the data value inside the javascript scope, but it doesn't update the DOM.
It's mentioned in the [docs](https://api.jquery.com/data/)
So even though the value in JS for `data-field` changes, in DOM, it still stays same, and I felt might cause problems in the future for cross-scopes.